record-layer_test.go

```diff
@@ -218,14 +217,14 @@ func TestEncryptRecord(t *testing.T) {
 	assertError(t, err, "Allowed a too-large record")
 }
 
-func TestReadWrite(t *testing.T) {
+func TestReadWriteTls(t *testing.T) {
```

Comment:
`Tls` -> `TLS`

I'll assume from here on that you'll just `s/Tls/TLS/g; s/Dtls/DTLS/g`